lib/es/management/commands/reindex.py

```diff
@@ -31,22 +31,34 @@
 
 # The subset of settings.ES_INDEXES we are concerned with.
 ES_INDEXES = settings.ES_INDEXES
+
+# We have this mapping so we can pass which indexer to use to celery.
+INDEXER_MAP = {
+    'webapp': WebappIndexer,
```

Comment:
Can we not hardcode these keys somehow ? I feel like we have too much stuff to modify when adding an index, between settings, this dict, the `INDEXES` tuple below, the indexer itself...